docs/Earn/bridge/bridge-mechanics.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Ankr Bridge Mechahics
+id: bridge-mechanics
+---
+
+# Ankr Bridge mechahics
+
+# Briding workflow
+
+The mechanics of what's going on under the hood when a user transfers their funds between network can be described as following:
+
+1. The user has Ankr Liquid Staking tokens on the network where our Staking smart contract minted them for said user.
```

Comment:
....smart contract is minted for the user.

Reply:
Erm.., no, literally a smart contract minted tokens for the user/to the user's address